src/app/(proper_react)/redesign/(authenticated)/user/welcome/EnterInfo.module.scss

```diff
@@ -85,6 +85,47 @@
   @media screen and (max-width: $screen-lg) {
     grid-template-columns: 1fr;
   }
+
+  input {
+    min-height: 3rem;
+  }
+
+  input[type="date"] {
+    background: transparent;
+
+    &::-webkit-date-and-time-value {
+      text-align: left;
+    }
+
+    &::before {
+      position: absolute;
+    }
```

Comment:
Could you add some comments explaining what these are for? I have no idea right now :sweat:

Reply:
Added comments in [e691fcc](https://github.com/mozilla/blurts-server/pull/3468/commits/e691fccfd105002a6ef7f1d8ba9c57ac12e12298).